### DISCUSSION TITLE
For left handed person

### DISCUSSION THREAD

**Discussion by Mahabanyabaramilta** [praydog/REFramework#297] [General Questions]
I was so exited to download your VR mods espacially ResidentEvil2 VR, But I couldn't enjoy it cause I'm left handed and It doesn't suppot it. So i want you to add left handed option for works you did and you will. I'm donating monthly for it. Not big money though I would be so happy if you add this option. Thank  you. I made github id for text it to you.

**Comment by NozzyEz** [comment]
Totally agree... being able to swap hands and thumbsticks would make these games infinitely more enjoyable... Don't get me wrong, they are already great!
